bugwarrior/services/phab.py

```diff
@@ -97,13 +133,13 @@ def issues(self):
 
         log.info("Found %i issues" % len(issues))
 
+        projects = fetch_issue_metadata(self.api, issues)
+
         for phid, issue in issues:
 
-            project = self.target  # a sensible default
-            try:
-                project = projects.get(issue['projectPHIDs'][0], project)
-            except IndexError:
-                pass
+            # TODO fix this now that we have access to proper projects.
+            # It was bugged before
+            project = self.target
```

Comment:
Cool. Could you make the comment more verbose though, to explain what needs fixing?